src/commands/crashlytics-mappingfile-upload.ts

```diff
@@ -0,0 +1,63 @@
+import { Command } from "../command";
+import { FirebaseError } from "../error";
+import * as utils from "../utils";
+
+import { fetchBuildtoolsJar, runBuildtoolsCommand } from "../crashlytics/buildToolsJarHelper";
+
+interface CommandOptions {
+  app: string | null;
+  mappingFile: string | null;
+  resourceFile: string | null;
+  debug: boolean | null;
+}
+
+interface JarOptions {
+  app: string;
+  mappingFilePath: string;
+  resourceFilePath: string;
+}
+
+export default new Command("crashlytics:mappingfile:upload <mappingFile>")
+  .description("Upload a mapping file to deobfuscate stack traces.")
+  .option("--app <appID>", "The app id of your Firebase app")
+  .option(
+    "--resource-file <resourceFile>",
```

Comment:
Is there an error if the resource file path isn't valid?

Reply:
I added a check in the calling function, in the next commit.